src/main/java/org/apache/sysml/hops/rewrite/RewriteRemovePersistentReadWrite.java

```diff
@@ -27,8 +27,9 @@
 import org.apache.sysml.hops.DataOp;
 import org.apache.sysml.hops.Hop;
 import org.apache.sysml.hops.Hop.DataOpTypes;
-import org.apache.sysml.hops.HopsException;
```

Comment:
I believe this file has been pulled in from a previous commit due to merging as well.